operator/autoscaler_test.go

```diff
@@ -272,8 +272,8 @@ func TestIncreaseShardToNodeRatioMore(t *testing.T) {
 
 	// scale-down even if this means increasing shard-to-node ratio of more than +1
 	esIndices := map[string]ESIndex{
-		"ad1": {Replicas: 0, Primaries: 21, Index: "ad1"},
-		"ad2": {Replicas: 0, Primaries: 2, Index: "ad2"},
+		"ad1": {Replicas: 1, Primaries: 21, Index: "ad1"},
```

Comment:
I prefer adding `eds.Spec.Scaling.MinIndexReplicas = 0` to this test instead of changing the index replicas to not alter the actual test spec.